### DISCUSSION TITLE
x-component-props传参    btnAbleStatus变为true，Buttons内disabled一直为false

### DISCUSSION THREAD

**Discussion by jxp1147** [alibaba/formily#2697] [Q&A 2.x] [answered]
<SchemaField.Void
    x-component="Buttons"
    x-component-props={{
         disabled: btnAbleStatus,
     }}
/>

 Buttons: (props) => {
        let { disabled } = props;
        console.log('btnAble', disabled, btnAbleStatus);    // btnAble false true

**Comment by janryWang** [comment, marked as answer]
https://formilyjs.org/zh-CN/guide/advanced/controlled
